docs/gettingStarted/setupAlchemyDevMode.md

```diff
@@ -1 +1,168 @@
+  - writing/modifying state of [Arc contracts](../../stack/arcIntro/)
+
+  Interaction of Alchemy with rest of the stack
+  ![Interaction](../images/daostack-interaction.png)
+
+  Following is the guide to start developing with alchemy if you are using already supported schemes by `client.js` and `subgraph`. If you have created your own `scheme contracts` for your DAO, please refer to [Add Custom Scheme support to Alchemy](../customSchemeSupportAlchemy)
+
+## Boilerplate
+
+    git clone https://github.com/daostack/alchemy.git
+    cd alchemy
+    npm ci
+
+## Setup Alchemy with Ganache (mode: development)
+
+    docker-compose build
+    docker-compose up -d
+
+  The above commands will build docker images and start the following services locally:
+
+  - **alchemy** => react frontend with webpack-dev-server
+  - **alchemy-server** => for storing proposal information for quick access
+  - **graph-node** => for handling events from blockchain as described in subgraph
+  - **ganache** => dev blockchain with some test DAOs deployed and loaded with GEN and Eth
+  - **subgraph-ipfs** => subgraph mappings on ipfs node
+  - **subgraph-postgres** => db for caching events based on subgraph and later fetched via GraphQL
+  - **redis**
+  - **alchemy-postgres**
+
+  Import test accounts that are setup with GEN and ETH to your metamask. You can get the account details by:
+
+    docker logs alchemy_ganache_1 | head -35
+
+
+  Now your playground is ready for developing.
+
+  TODO: Currently webpack does not detect changes in all components and rebuilds only if top-level `src/file` is changed. For now you can touch the any file in top-level and this should trigger rebuild
+
+  NOTE:
+
+  1. If the feature integration requires you to interact with outside contracts (e.g. uniswap widget integration might require uniswap contracts), then you can simply deploy those contracts to same `ganache` container using truffle or your own deployment script.
+  2. See `Client.js` documentation for more integration details
+
+## Setup Alchemy with Testnet (mode: staging)
+
+  Often Ganache does not behave same as production. If you want to setup Alchemy for interacting with testnet and check before you submit PR,
+  then after the boilerplate steps -
+  
+
+  Choose from one of the following setup for testnet to start playing/integrating features to Alchemy:
+  
+  - [Use daostack rinkeby subgraph](#use-daostack-rinkeby-subgraph)
+  - [Run graph-node locally](#run-graph-node-locally)
+
+  
+  NOTE: 
+  
+  1. Alchemy only shows `daos` that are registered via `DAOregistry` and tracked by daostack subgraph for the respective network. You can send the `.json` of your DAO details to us (contact Nave Rachman, telegram: @NaveRachman) and we will help you.
+
+  2. Since above process of registering DAO takes up to 24hrs in following section we provide way to hack it during development
+
+### Use daostack rinkeby subgraph
+
+  Choose this if,
```

Comment:
There is actually an easier way. If you run against rinkbey (and the rinkeby subgraph) there is no need to run any docker containers at all, but just run the react server directly with setting NODE_ENV to staging. 

There' s a shortcut in the package.json file:

`npm run start-staging`

(and tehre is also `npm run start-prod` to run with main-net settings)

Reply:
So redis and postgres also runs outside docker?